Sources/DeepLearning/Operators/NN.swift

```diff
@@ -65,45 +79,15 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
-public extension Tensor where Scalar: BinaryFloatingPoint {
-    /// Computes the batch normalized tensor along the specified axis.
-    ///
-    /// Specifically, returns `(self - mu)/(var + epsilon) * gamma + beta` where
-    /// `mu` and `var` are respectively the mean and variance of `self` along
-    /// `axis`.
-    ///
-    /// - Parameters:
-    ///     - axis: The batch dimension.
-    ///     - offset: The offset, also known as beta.
-    ///     - scale: The scale, also known as gamma.
-    ///     - epsilon: A small value added to the denominator for numerical
-    ///         stability.
-    @inlinable
-    @differentiable(
-        wrt: (self, offset, scale), vjp: _vjpBatchNormalized
-        where Scalar : TensorFlowFloatingPoint
-    )
-    func batchNormalized(
-        alongAxis axis: Int32,
-        offset: Tensor = Tensor(0),
-        scale: Tensor = Tensor(1),
-        epsilon: Scalar = 0.001
-    ) -> Tensor {
-        let mean = self.mean(alongAxes: axis)
-        let squaredDiff: Tensor = Raw.squaredDifference(self, mean)
-        let variance = squaredDiff.mean(alongAxes: axis)
-        let inv = rsqrt(variance + epsilon) * scale
-        return self * inv + offset - mean * inv
-    }
-}
-
 //===------------------------------------------------------------------------------------------===//
-// Convolution and pooling
+// Convolution and Pooling
 //===------------------------------------------------------------------------------------------===//
 
 /// A padding scheme. Used by padding, convolution, and pooling ops.
 // @_frozen // SR-9739
 public enum Padding {
+    /// The "explicit" padding scheme.
+    case explicit(_ paddings: [Int32])
```

Comment:
Enum associated value lists are different from function parameter lists. This can just be a type, instead of `_ label:`.

```suggestion
    case explicit([Int32])
```

Reply:
That makes sense. I fixed this both here and in the other PR. :)